tmpl/wiki/package-hardening.md

```diff
@@ -0,0 +1,118 @@
+## Package hardening
```

Comment:
as mort mentioned on mirageos-devel, `hardening` has security connotations, whereas this is more about 'towards high-quality packages' (and not only about security)